pro/src/components/layout/CsvTable/CsvTableRoute.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FunctionComponent } from 'react'
+
+import HeaderContainer from '../Header/HeaderContainer'
```

Comment:
autant que possible, eviter les import relatif.

```suggestion
import HeaderContainer from 'components/layout/Header/HeaderContainer'
```

Par ailleurs, l'ancien `CsvTableContainer` allait render le composant `CsvTable`, je ne pense pas que render le `Header` suffise à render la page.